src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -1,15 +1,35 @@
 package github.weichware10.toolbox;
 
 import github.weichware10.toolbox.gui.ConfirmBox;
-import github.weichware10.toolbox.gui.Startbildschirm;
+import github.weichware10.toolbox.gui.TestVorbildschirm;
+import github.weichware10.util.Logger;
+import github.weichware10.util.config.CodeChartsConfiguration;
+import github.weichware10.util.config.ConfigClient;
+import github.weichware10.util.config.Configuration;
+import github.weichware10.util.config.ZoomMapsConfiguration;
+import github.weichware10.util.db.DataBaseClient;
+import io.github.cdimascio.dotenv.Dotenv;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
+
 import javafx.application.Application;
+import javafx.event.ActionEvent;
 import javafx.event.EventHandler;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
 import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ButtonBar.ButtonData;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.control.ButtonBar.ButtonData' import. Should be before 'javafx.scene.control.TextField'.